### PR TITLE
fix: corregir diseño añadir socio

### LINKED ISSUE #184: Corregir el diseño del formulario "Añadir socio" y sus funciones
### EXPLANATION 

Si comparamos el diseño de la vista "Añadir socio" se puede notar que no coincide con el diseño asignado, hay que centrar mas el el marco que quede mas centrado .

Falta agregar la opción regresar que esta en la parte superior de la pantalla y  su función sera volver a la vista anterior (Socio).
falta la función de las opciones   "Añadir"  "Eliminar" y su función correspondiente.

Eliminar su  función , al darle clic a eliminar este te mostrar una vista con un mensaje de comfirmacion que te pregunta si deseas eliminar

Añadir su función, al ver llenado todos los datos y darle clic en añadir guardara la información agregandote al grupo de socios

#### SCREENSHOTS

Diseño actual. No tiene la opción regresar centrar mas el marco de la informacion
image

Diseño asignado

image

Diseño asignado. Al darle clic eliminar mostrara la siguiente vista con un mensaje de comfirmacion

image

### USER STORIES TO FULFILL

Como usuario quiero que  el espacio de la información este mas centrado

como usuario quiero que la opción Añadir funcione guardando la información
 
como usuario quiero que la opción Eliminar funcione como esta estimulada

como usuario quiero que este la opción regresar y  funcione llevando al a vista anterior (socio)

## Dependencies

- https://github.com/Azordev/did-admin-panel/issues/132 The modal should work in order to verify the deletion

# Referencias

- Por favor terminar en 3 días. Mas información: https://github.com/Azordev/did-admin-panel/issues/192

### PR DESCRIPTION
**Pull request by JuanParraIV**
### Description

Please include a summary of the change.

fixes # (#184 )

#### Screenshots
image
image
image
image

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
Please rebase

**Comment by e-azocar** [comment]
Please check this
image

**Review by e-azocar — APPROVED** [review]

File: views/Users/Create/Form.module.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  overflow: hidden;
+  position: absolute;
+  width: 0.1px;
+  z-index: -1;
+}
+
+.label,
+.label-title {
+  color: $gray-300;
+  font-family: 'Visby', sans-serif;
```
Comment (Israel-Laguan) [inline views/Users/Create/Form.module.scss]: Change to variables

**Review by Israel-Laguan — COMMENTED** [review]
Remove yarn lock please

**Comment by Israel-Laguan** [comment]
Checked that "Anhadir Socio" form looks better, further improvements will be in another issue

**Pull request merged**